### ISSUE TITLE
ADFS integration with OS

### ISSUE THREAD

**Issue by brandonopened** [opensalt/opensalt#542]
- [ ] ADFS:  Add UI on login to click to enable ADFS
    Need  complete related tasks before we can point this story.
    
     
    
    As an OS user  
     I want the system updated  
     So that it uses ADFS for login/single sign on
    
    **SCOPE**  
     The scope of this story is to add the "Login using Active Directory" option on the UI login page to click to enable ADFS.  The regular login is still available but should be a different user name.  See mockup in Documentation section for more information
    
    **OUT OF SCOPE**  
     Install compatible backend layer for Symfony to enable SAML will be completed with KMS-4354  
     Create configurability of SAML/Active Directory will be completed with KMS-4362  
     Configure AD to use OpenSALT user permission structure will be completed with KMS-4370
    
    **AUTOMATED ACCEPTANCE TEST CASES**  
    <need input from team>
    
    **DOMAIN**  
    <need input from team> 
    
    **DOCUMENTATION** [https://opensalt.mybalsamiq.com/projects/opensalt/Login%20Page](https://opensalt.mybalsamiq.com/projects/opensalt/Login%20Page)
    

Created from Aha! https://act.aha.io/features/ACKM-313

**Comment by roverwolf** [comment]
Related to #502 ?
